views/jira-manual-app-creation.hbs

```diff
@@ -108,26 +108,26 @@
 
         <div class="field-group ">
           <label for="app-id">GitHub app ID</label>
-          <input class="text full-width-field" type="text"
+          <input class="text full-width-field" type="number"
                  data-aui-validation-field required
                  value="{{app.appId}}"
-                 id="app-id" name="app-id" />
+                 name="appId" />
         </div>
 
         <div class="field-group">
           <label for="client-id">GitHub client ID</label>
-          <input class="text full-width-field" type="text"
+          <input class="text full-width-field" type="number"
```

Comment:
This needs to be a string otherwise I can't enter my client id in the format of `Iv1.<mixture_of_numbers+letters>`

Reply:
updated to text